hax/hax/util.py

```diff
@@ -2137,11 +2139,11 @@ def processEventToObjHealth(self, proc_type: m0HaProcessType,
 
     def service_health_to_m0dstatus_update(self, proc_fid: Fid,
                                            svc_health: ObjHealth):
-        ev = ConfHaProcess(chp_event=self.objHealthToProcessEvent(svc_health),
-                           chp_type=int(
-                                m0HaProcessType.M0_CONF_HA_PROCESS_M0D),
-                           chp_pid=0,
-                           fid=proc_fid)
+        ev = ConfHaProcess(
+            chp_event=self.objHealthToProcessEvent(svc_health),
```

Comment:
Please avoid making unrelated changes. If there's a regression then it becomes difficult to map the changes to the problem.

Reply:
Also less changes, less time to review.